spike-worker/generate-census-records.js

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+'use strict'
+
+const csv = require('fast-csv')
+const fs = require('fs')
+const faker = require('faker')
+const moment = require('moment')
+const commandLineArgs = require('command-line-args')
+const upnService = require('../admin/services/upn.service')
+const outputFilename = 'pupilCensusData.csv'
+
+const optionDefinitions = [
+  { name: 'recordsLength', alias: 'r', type: Number }
+]
+
+const options = commandLineArgs(optionDefinitions)
+
+function writeCsv (data) {
+  if (!data.length) {
+    return
+  }
+  const headers = ['LEA', 'Estab', 'UPN', 'Surname', 'Forename', 'Middlenames', 'Gender', 'DOB']
+  const ws = fs.createWriteStream(outputFilename, { flags: 'a' })
+  csv
+    .write(data, {headers: headers})
+    .pipe(ws)
+  ws.write('\n')
+}
+
+async function main (options) {
```

Comment:
Does this need to be async?  I don't see any `await`s

Reply:
@jon-shipley I forgot to remove it, thanks.